content/release-notes/1.61.0.md

```diff
@@ -0,0 +1,23 @@
+---
+date: 2022-01-28T10:21:28-07:00
+linktitle: "1.61.0"
+title: "1.61.0"
+kubernetes: "1.20, 1.21, and 1.22"
+---
+
+{{<features>}}
+* Adds a CLI command to collect all versions for an application. See [the docs](https://kots.io/kots-cli/get/versions/) for more details.
+* Adds the ability to block installing a certain application if the current KOTS version is incompatible with the KOTS version required by the app.
+* Adds the ability to block upgrading the application if the current KOTS version is incompatible with the KOTS version required by the release.
+* Adds functionality to display a modal with commands for upgrading KOTS if there the version attempting to be deployed has a different KOTS version specified than the one the user has installed.
+{{</features>}}
+
+{{<changes>}}
+
+{{</changes>}}
+
+{{<fixes>}}
+* Fixes a bug that caused images to be pushed to private registry multiple times during airgap install.
+* Fixes a bug that displayed a message to "edit the current config when performing a new install
```

Comment:
I don't think the quotes are neceesary here (they also do not close or have proper punctuation to end the sentence)I think this should be something more like the following:
* Fixes a bug that erroneously displays a message to edit the current config when performing a new install.

Alternatively:
* Fixes a bug that erroneously displays a message to edit the current config when performing a new install where a current config does not exist.